keadm/cmd/keadm/app/keadm.go

```diff
@@ -28,7 +28,6 @@ import (
 //Run executes the keadm command
 func Run() error {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
-	pflag.Set("logtostderr", "true")
```

Comment:
why remove this line?

Reply:
It always returns err so it does not do impact :(

Reply:
sorry, I am not quite follow you, but consider the default value of `logtostderr` is true I am fine with delete this line. 
question is that change is not related with the errcheck linter, right?